pcf26536/politico-api
# #163901869 - Added database configurations

**@johnmusiu, I have completed chore #163901869, Setting up PostgresSQL DB Configurations**

**What does this PR do?**
Sets up PostgresSQL DB configurations files and models for API v2

**Description of Task to be completed?**
- Creating a database user
- Creating a database
- Setting the database configurations in the travis.yml and environment
- Creating tables
- Adding database transactions model
- Adding a base model skeleton for other models

**How should this be manually tested?**
To test this manually, clone the repo to your local system and run the applications. You can check the log for any output from the process

**Prerequisites**
An installed and running instance of PostgreSQL

What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163901869
